package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "linter-gjslint",
   "main": "./lib/init",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
Don't update the version manually, `apm publish` will do this when the update is pushed up to Atom's package manager.